fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -161,6 +162,7 @@ NameOffMRI=`getopt1 "--fmriname" $@`
 SubjectFolder=`getopt1 "--subjectfolder" $@`
 BiasCorrection=`getopt1 "--biascorrection" $@`
 UseJacobian=`getopt1 "--usejacobian" $@`
+PreregisterTool=`getopt1 "--preregistertool" $@`
 
 if [[ -n $HCPPIPEDEBUG ]]
 then
```

Comment:
Another hidden "debugging" type feature.  Harmless I guess, but not consistently implemented.

Reply:
Another for cleanup. That was there before.